src/Inventory.java

```diff
@@ -0,0 +1,153 @@
+            } else {
+                System.out.println("You cannot sell more than you have.");
+            }
+
+        } else {
+            System.out.println("The item name you entered does not exist.");
+        }
+    }
+
+    public int menuOptions() {
+        int option = 0;
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("[1] Create a new Item." + System.lineSeparator() + "[2] Sell some items." + System.lineSeparator() + "[3] Remove an Item." + System.lineSeparator() + "[4] Update an Item's Quantity." + System.lineSeparator() + "[5] Update an Item's Price." + System.lineSeparator() + "[6] Quit Program." + System.lineSeparator());
+
+        option = scanner.nextInt();
+
+        return option;
+    }
+
+    public Item findItem(String name) {
+        Item needItem = null;
+
+        for (Item foundItem : this.items) {
+            if (foundItem.getName().equalsIgnoreCase(name)) {
+                needItem = foundItem;
+            }
+        }
+        return needItem;
+    }
+
+    public String findItemName(String name) {
+        for (Item item : this.items) {
+            if (item.getName().equalsIgnoreCase(name)) {
+                name = item.getName();
+            }
+        }
+        return name;
+    }
+
+    public void sellItem(Item sellItem, int sellAmount) {
+        int amount = sellItem.getQuantity();
+        amount -= sellAmount;
+        sellItem.setQuantity(amount);
+    }
+
+    public void removeItem() {
+        int index = -1;
+        System.out.println("Please enter the 'NUMBER' of the item you'd like to remove: ");
+        int idToRemove = scanner.nextInt();
+        for (int i = 0; i <= items.size(); i++){
+            Item current = items.get(i);
+            if (current.getId() == idToRemove) {
+                items.remove(i);
+                System.out.println("Yay! Successful removal!");
+            }
+        }
+    }
+
+    public void changeQuantity(){
+        System.out.println("Enter the name of the item you would like to update: ");
+        String itemName = scanner.nextLine();
+        if (itemName.equalsIgnoreCase(findItemName(itemName))){
+            scanner.nextLine();
+            System.out.println("What would you like to update the quantity to: ");
+            int newQuantity = scanner.nextInt();
+            Item item = findItem(itemName);
+            item.setQuantity(newQuantity);
+            System.out.println("Your new quantity is " + item.getQuantity());
+        }else{
+            System.out.println("The name you entered is invalid.");
+        }
+
+    }
+
+    public void changePrice(){
+        System.out.println("Enter the name of the item you would like to update: ");
+        String itemName = scanner.nextLine();
+        scanner.nextLine();
+        if (itemName.equalsIgnoreCase(findItemName(itemName))) {
+            System.out.println("What would you like to update the price to: ");
+            int newPrice = scanner.nextInt();
```

Comment:
`int` probably isn't the best type for money unless you never want a sale with cents.